source/fortran/iteration_variables.f90

```diff
@@ -3377,7 +3377,7 @@ subroutine init_itv_152
     implicit none
     lablxc(152) = 'fgwsep        '
     boundl(152) = 0.001D0
-    boundu(152) = 1.000D0
+    boundu(152) = 0.1D0
   end subroutine init_itv_152
```

Comment:
I am a little concerned about changing the default bounds as this could lead to existing input files (that use the default bounds) starting outside of the feasible region. In this instance, the 3 changes are minor so I think I am happy with it.

Reply:
The output values in the large tokamak regression are 0.85 and 0.5 for pedestal and separatrix respectively. @mkovari 
Which is outside the bounds recommended

Reply:
Electron density at separatrix / nGW                                     (fgwsep_out)              5.000E-01
Electron density at pedestal / nGW                                       (fgwped_out)              8.500E-01

From the output file of the large tokamak regression test

Reply:
I would still like the warning to be explicit about what sort of lower and upper bounds people should set.

For example:
dene Recommended value of boundl(6) > 80% of expected volume-averaged density.

Reply:
Is this true for all cases? Even for a stellerator?
The output warning are limited to one line so I will need to amend the current one as I cant add it under unless we change their length

Reply:
Maybe point people to a section in the documentation with the warning? And have a longer explanation in the docs.